p2p/host/autorelay/relay_finder.go

```diff
@@ -194,41 +204,91 @@ func (rf *relayFinder) background(ctx context.Context) {
 	}
 }
 
+func (rf *relayFinder) runScheduledWork(ctx context.Context, now time.Time, scheduledWork *scheduledWorkTimes, canCallPeerSourceChan chan<- struct{}) bool {
```

Comment:
Nit: Any reason to pass `now` to this function instead of just calling `Now()` first thing here?

Reply:
pureness